nncf/quantization/algorithms/weight_compression/onnx_backend.py

```diff
@@ -328,3 +368,94 @@ def _add_dequantize_linear_layer(
         # Update the node mapping
         self.name_to_node_map[dequantize_node.name] = dequantize_node
         return model
+
+    def _replace_matmul_with_matmulnbits(
+        self,
+        model: onnx.ModelProto,
+        weight_compression_parameters: WeightCompressionParameters,
+        quantized_weights: np.ndarray,
+        orig_weight,
+        scale: np.ndarray,
+        zero_point: Optional[np.ndarray],
+        axis: int,
+        block_size: int,
+        weight_name: str,
+        weight_dtype: onnx.TensorProto.DataType,
+        scale_dtype: onnx.TensorProto.DataType = onnx.TensorProto.FLOAT,
+    ):
+        if weight_dtype == onnx.TensorProto.INT4:
```

Comment:
Please add else block with rising error

Reply:
I guess we should raise an error when `weight_type` is not in [`onnx.TensorProto.INT4`, `onnx.TensorProto.UINT4`]. We just shouldn't pack to uint8 in that case.